docs/reference/client.md

```diff
@@ -0,0 +1,9 @@
+(client_side_components)=
+
+# Client-side components
+
+```{include} ../../client/README.md
+:end-before: <!-- END -->
+```
+
+[Proceed to the client-side API documentation.](./client/index.html){.external}
```

Comment:
This allows us to reuse the same paragraphs for the intro in both the Sphinx docs and the Typedoc docs, but with different links to navigate to each other.

Ref: https://mystmd.org/guide/embed#docs-include